src/test/java/no/nav/veilarbarena/config/KafkaTestConfig.java

```diff
@@ -1,87 +1,96 @@
 package no.nav.veilarbarena.config;
 
-import no.nav.veilarbarena.kafka.KafkaHelsesjekk;
-import no.nav.veilarbarena.kafka.KafkaProducer;
-import no.nav.veilarbarena.kafka.KafkaTopics;
-import org.apache.kafka.clients.producer.ProducerConfig;
-import org.apache.kafka.common.serialization.StringDeserializer;
-import org.apache.kafka.common.serialization.StringSerializer;
-import org.springframework.beans.factory.annotation.Autowired;
+import io.micrometer.core.instrument.MeterRegistry;
```

Comment:
Kunne unngått egen KafkaConfig for test, (ref. veilarbvedtaksstotte)

Reply:
Tenkte på det da jeg satt opp configen, droppet det siden det ikke var så mye duplisert kafka oppsett, men kanskje det er like greit å sette det opp nå før det vokser ut i fremtiden

Reply:
Tenker det ikke er veldig farlig nå.

Reply:
Sikkert ikke så farlig nei, men legger det til uansett :)